src/archivematicaCommon/lib/fileOperations.py

```diff
@@ -155,20 +155,21 @@ def updateFileLocation2(src, dst, unitPath, unitIdentifier, unitIdentifierType,
     srcDB = src.replace(unitPath, unitPathReplaceWith)
     dstDB = dst.replace(unitPath, unitPathReplaceWith)
     # Fetch the file UUID
-    sql = """SELECT Files.fileUUID, Files.currentLocation FROM Files WHERE removedTime = 0 AND Files.currentLocation = '%s' AND %s = '%s';""" % (MySQLdb.escape_string(srcDB) , unitIdentifierType, unitIdentifier)
-    rows = databaseInterface.queryAllSQL(sql)
-    if len(rows) != 1:
-        print >> sys.stderr, 'ERROR: file information not found:', len(rows), "rows for SQL:", sql
-        print >> sys.stderr, 'Rows returned:', rows
+    kwargs = {
+        "removedtime__isnull": True,
+        "currentlocation": srcDB,
+        unitIdentifierType: unitIdentifier
+    }
+    files = File.objects.filter(**kwargs)
+    count = files.count()
+    if count != 1:
+        print >> sys.stderr, 'ERROR: file information not found:', count, "rows for arguments:", repr(kwargs)
```

Comment:
Use .get() and catch DoesNotExist and MultipleResults instead of counting?  Does mean you can't use .update below, but I think it's clearer.